src/components/Select/Select.scss

```diff
@@ -41,21 +41,6 @@ $stacking-order: (
   }
 }
 
-.placeholder {
-  // stylelint-disable-next-line selector-max-class
-  &.error .Input {
-    // This is the only place this color is used.
-    // stylelint-disable-next-line color-no-hex
-    color: #9c9798;
-  }
-
-  // stylelint-disable-next-line selector-max-class, selector-max-specificity
-  &.error .Input:-moz-focusring {
-    color: transparent;
-    text-shadow: var(--p-override-none);
-  }
-}
```

Comment:
Removing unused class. Here's [the legacy commit from when it was added](https://github.com/Shopify/polaris-react/blob/daaa92ec1ff45a5ebc8f1d6cc431ce259368d6e3/src/components/Select/Select.tsx#L68)

Reply:
Amazing! Thanks for the context Laura. I think it makes sense to remove this code.